### DISCUSSION TITLE
Runelite Highscores - Removing Unranked stats

### DISCUSSION THREAD

**Discussion by osrs-invalid** [runelite/runelite#18062] [Q&A] [answered]
If anyone knows how to remove their unranked high scores from runelite it'd really help.

When I log out of that account for a while it dissapears but soon after I log in the runelite highscores almost immediately update to display that unranked stat.

I've tried disabling Wiki Sync but that has not helped.

**Comment by Bram91** [comment, marked as answer]
That feature relies on the OSRS highscores; we just show the data. As such, it is not possible to turn it off/change its behavior.

**Reply by osrs-invalid** [reply]
> That feature relies on the OSRS highscores; we just show the data. As such, it is not possible to turn it off/change its behavior.

Appreciate the response. But that cannot be true because the official OSRS highscores page does not display unranked stats.

Another way to back that it is not correct is that only after I log in does the stat update to re-display.

Based on that. It must be something within runelite that is syncing to display this. The question is where can I find that function and can it be turned off?

**Reply by YvesW** [reply]
The HiScore plugin only pulls stats from the official HiScores. You can read the code here: https://github.com/runelite/runelite/tree/master/runelite-client/src/main/java/net/runelite/client/plugins/hiscore and https://github.com/runelite/runelite/tree/master/runelite-client/src/main/java/net/runelite/client/hiscore
Bram is correct.

**Reply by osrs-invalid** [reply]
> The HiScore plugin only pulls stats from the official HiScores. You can read the code here: https://github.com/runelite/runelite/tree/master/runelite-client/src/main/java/net/runelite/client/hiscore
> 
> Bram is correct.

The official high scores do not display unranked stats. 

If I do not play runelite, my unranked stat does not display.

Something is being pulled from my account while using runelite. As that unranked stat only appears on the runelite highscores after I have logged in via runelite.

This stat is not available via the official high scores page.

Edit: is this related https://github.com/runelite/runelite/pull/6829

**Reply by osrs-invalid** [reply]
> > The HiScore plugin only pulls stats from the official HiScores. You can read the code here: https://github.com/runelite/runelite/tree/master/runelite-client/src/main/java/net/runelite/client/hiscore
> > 
> > Bram is correct.
> 
> The official high scores do not display unranked stats. 
> 
> If I do not play runelite, my unranked stat does not display.
> 
> Something is being pulled from my account while using runelite. As that unranked stat only appears on the runelite highscores after I have logged in via runelite.
> 
> This stat is not available via the official high scores page.
> 
> Edit: is this related https://github.com/runelite/runelite/pull/6829

There appears to be both a public and private pull:

	private String player;
	private Map<HiscoreSkill,

Is that right?

**Reply by osrs-invalid** [reply]
Is it building something?


skills.put(s, new Skill(activity.rank, (int) activity.score, -1L));

return new HiscoreResult(username, skills.build());

**Comment by osrs-invalid** [comment]
> That feature relies on the OSRS highscores; we just show the data. As such, it is not possible to turn it off/change its behavior.

Please see my other replies

**Comment by cdfisher** [comment]
Are you using the plugin "Unranked KC Tracker" from the Plugin Hub?

**Reply by osrs-invalid** [reply]
> Are you using the plugin "Unranked KC Tracker" from the Plugin Hub?

Really appreciate the response but unfortunately that's not added as a plugin on my runelite.

If you have any other ideas please please do let me know!

**Reply by cdfisher** [reply]
Can you attach your `client.log` file? Instructions on where to find it are [here](https://github.com/runelite/runelite/wiki/FAQ#where-can-i-find-the-logs-screenshots-or-configurations).

**Reply by osrs-invalid** [reply]
> Can you attach your `client.log` file? Instructions on where to find it are [here](https://github.com/runelite/runelite/wiki/FAQ#where-can-i-find-the-logs-screenshots-or-configurations).

I've found it but unable to send, it says invalid format. Only photos are allowed here?

I can upload here if you can let me know where to please. 

Really appreciate your help.
